geopandas/_geoseries.pyx

```diff
@@ -0,0 +1,105 @@
+from warnings import warn
+
+import shapely
+from shapely.geometry import MultiPoint, MultiLineString, MultiPolygon
+from shapely.geometry.base import BaseGeometry
+from shapely.ops import cascaded_union, unary_union
+import shapely.affinity as affinity
+
+import cython
+cimport cpython.array
+
+cimport numpy as np
+import numpy as np
+import pandas as pd
+from pandas import Series, DataFrame, MultiIndex
+
+import geopandas as gpd
+
+from .base import GeoPandasBase
+
+include "_geos.pxi"
+
+
+def _series_op(this, other, op, **kwargs):
+    if kwargs:
```

Comment:
Not just that, but many operators that are not currently supported.  Currently this is handled by the `NotImplementedError` try-except block below.  We would want to extend this approach to all relevant operations before considering merging though (if we want to consider that at all).